lib/picos/ocaml5/picos_ocaml.ml

```diff
@@ -63,7 +63,7 @@ module Handler = struct
       Some
         (fun k ->
           match h.yield c with
-          | unit -> Effect.Deep.continue k unit
+          | () -> Effect.Deep.continue k ()
```

Comment:
This was actually intentional (potentially generating a shorter instruction, i.e. a register to register move), but it doesn't really matter. 😄

Reply:
hu, does the codegen change?? :cold_sweat: I expect a match against `()` to be fully skipped…